storage_service/locations/models/space.py

```diff
@@ -145,12 +145,13 @@ class Space(models.Model):
     NFS = "NFS"
     OFFLINE_REPLICA_STAGING = "REPLICA"
     PIPELINE_LOCAL_FS = "PIPE_FS"
+    RCLONE = "RCLONE"
     SWIFT = "SWIFT"
     GPG = "GPG"
     S3 = "S3"
     # These will not be displayed in the Space Create GUI (see locations/forms.py)
     BETA_PROTOCOLS = {}
-    OBJECT_STORAGE = {DATAVERSE, DSPACE, DSPACE_REST, DURACLOUD, SWIFT, S3}
+    OBJECT_STORAGE = {DATAVERSE, DSPACE, DSPACE_REST, DURACLOUD, SWIFT, S3, RCLONE}
```

Comment:
Should rclone ve before Swift, to keep alphabetical order here?

Reply:
yes, good call!